src/navigation/RootNavigator/index.tsx

```diff
@@ -118,6 +122,7 @@ export type RootNavigatorParamList = {
 const Stack = createStackNavigator<RootNavigatorParamList>();
 
 const RootNavigator = () => {
+  const showUnlockApplicationScreen = useSetting('autoAppLock');
```

Comment:
What will happen when a user upgrades? Is this value going to be returned as `false` if the setting has no default value?

Reply:
On app upgrade this flag will default to `true` to not change the app behaviour. See [here](https://github.com/desmos-labs/dpm/blob/a004ffca79b19adfb314d9a7e1ac33c0e591867e/src/recoil/settings.ts#L23).